dallinger/experiment_server/dashboard.py

```diff
@@ -709,15 +718,30 @@ def prep_datatables_options(table_data):
 @dashboard.route("/database")
 @login_required
 def database():
+    from dallinger.db import get_polymorphic_mapping
     from dallinger.experiment_server.experiment_server import Experiment, session
 
-    title = "Database View"
     exp = Experiment(session)
-    model_type = request.args.get("model_type")
-    if model_type:
-        title = "Database View: {}s".format(model_type)
+
+    table = request.args.get("table", None)
+    polymorphic_identity = request.args.get("polymorphic_identity", None)
+
+    if polymorphic_identity == "None":
+        polymorphic_identity = None
+
+    if table is None and polymorphic_identity is None:
+        table = "participant"
+
+    if polymorphic_identity is not None:
+        assert table is not None
+        cls = get_polymorphic_mapping(table)[polymorphic_identity]
+        label = cls.__name__
+    else:
+        label = table.capitalize()
```

Comment:
Why is this? Is this just for readability on the front end?

Reply:
Yep, otherwise it looks inconsistent with the other tables, which are labelled with class names.